packages/opentelemetry-plugin-document-load/src/types.ts

```diff
@@ -0,0 +1,50 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { PerformanceTimingNames } from './enums/PerformanceTimingNames';
+
+/**
+ * Performance metrics
+ */
+export type PerformanceEntries = {
```

Comment:
nit: Could this use [`Partial`](https://www.typescriptlang.org/docs/handbook/utility-types.html#partialt)?